packages/embark-ui/src/components/ContractOverview.css

```diff
@@ -1,3 +1,36 @@
+.contract-function-badge {
```

Comment:
FYI, if you want, uou can convert this file in `.scss`, then you can use indentation scoping (probably not the real name). Here's an example:
```
.contract-function-badge {
  cursor: default;

  code, .contract-function-button {
    font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
    font-size: 13.1333px !important;
    font-weight: 700 !important;
  }
...
}
```
Everything within the brackets get prepended by `.contract-function-badge `

Reply:
I'm unfamiliar with SCSS (I know what it is, have never actually authored any or looked at the docs) but can give it a look. It does look like it would result in a better structuring of the styles.

Reply:
It's the best thing is the world. It makes writing CSS 5 times betters.
It,s super simple: https://sass-lang.com/guide
The part I talked about above is "Nesting"

Reply:
Thanks!

Reply:
I converted ContractLog.css and ContractOverview.css to SCSS and made use of nesting where possible without doing a total refactor. It could still probably be better, but it's a start!